policies/master_payer.json

```diff
@@ -29,16 +29,28 @@
     },
     {
       "Action": [
-        "aws-portal:View*",
+        "account:GetAccountInformation",
```

Comment:
NOTE: @muellerk22 and @silvexis, I made the changed described in https://cloudzero.atlassian.net/browse/CP-11100.